helm/mercure/templates/deployment.yaml

```diff
@@ -0,0 +1,103 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "mercure.fullname" . }}
```

Comment:
Personally, I suffix the resource name with the kind in lowercase as made here : https://helm.sh/docs/chart_template_guide/named_templates/#declaring-and-using-templates-with-define-and-template

Reply:
As it is the main deployment, the name would be weird or hard to define: `mercure-mercure`, `mercure-deployment`, etc.
In this special case, I don't recommend to specify a suffix

Reply:
I don't recommend that either.